sdk/keyvault/keyvault-certificates/tests-resources.json

```diff
@@ -51,16 +51,7 @@
                 "verify",
                 "wrapKey"
               ],
-              "secrets": [
-                "backup",
-                "delete",
-                "get",
-                "list",
-                "purge",
-                "recover",
-                "restore",
-                "set"
-              ],
+              "secrets": ["backup", "delete", "get", "list", "purge", "recover", "restore", "set"],
```

Comment:
nit: why do this change? It makes it different than the other arrays it's beside

Reply:
I'll undo this change